app.R

```diff
@@ -18,30 +22,58 @@ ui <- fluidPage(
     # Sidebar with a slider input for number of bins
     sidebarLayout(
         sidebarPanel(
+            checkboxGroupInput(inputId = "stems_plot",
+                               label = "Census year:",
+                               choices = c("2008-2010" = "stems_plot$CensusID == 1",
```

Comment:
Instead of setting `input$stems_plot` to values like `"stems_plot$CensusID == 1"` and `"stems_plot$CensusID == 1"`, let's set them to ultra simple `"1"` and `"2"`. Then using these inputs you can do a `filter(CensusID %in% input$stems_plot)`.